mne/label.py

```diff
@@ -747,6 +747,9 @@ def center_of_mass(self, subject=None, restrict_vertices=False,
         subject = _check_subject(self.subject, subject)
         if np.any(self.values < 0):
             raise ValueError('Cannot compute COM with negative values')
+        if np.all(self.values == 0):
+            raise RuntimeWarning('Label values are all 0, treating as a structural label\n')
```

Comment:
Ah, thanks, I'll change it up.  

Also, I noticed that the label class says that when the label.values don't exist they are set to ones.  But read_labels_from_annot currently sets them to zeros.  Should that be ones instead?